spec/models/profile_spec.rb

```diff
@@ -5,20 +5,20 @@
   it { should_not have_valid(:user_id).when(nil, '', 'asdf', 'dog') }
 
   it { should have_valid(:blog_url).when('http://www.google.com', 'https://wwww.google.com') }
-  it { should_not have_valid(:blog_url).when(nil, '', 'asdf', 'dog', 'www.google.com') }
+  it { should_not have_valid(:blog_url).when('asdf', 'dog', 'www.google.com') }
 
   it { should have_valid(:facebook).when('http://www.google.com', 'https://wwww.google.com') }
-  it { should_not have_valid(:facebook).when(nil, '', 'asdf', 'dog', 'www.google.com') }
+  it { should_not have_valid(:facebook).when('asdf', 'dog', 'www.google.com') }
 
   it { should have_valid(:twitter).when('http://www.google.com', 'https://wwww.google.com') }
-  it { should_not have_valid(:twitter).when(nil, '', 'asdf', 'dog', 'www.google.com') }
+  it { should_not have_valid(:twitter).when('asdf', 'dog', 'www.google.com') }
 
   it { should have_valid(:github).when('http://www.google.com', 'https://wwww.google.com') }
-  it { should_not have_valid(:github).when(nil, '', 'asdf', 'dog', 'www.google.com') }
+  it { should_not have_valid(:github).when('asdf', 'dog', 'www.google.com') }
 
   it { should have_valid(:linked_in).when('http://www.google.com', 'https://wwww.google.com') }
-  it { should_not have_valid(:linked_in).when(nil, '', 'asdf', 'dog', 'www.google.com') }
+  it { should_not have_valid(:linked_in).when('asdf', 'dog', 'www.google.com') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.